uni/lib/view/exams/exams.dart

```diff
@@ -102,35 +102,23 @@ class ExamsPageViewState extends GeneralPageViewState<ExamsPageView> {
       key: Key(keyValue),
       margin: const EdgeInsets.only(bottom: 8),
       padding: const EdgeInsets.all(8),
-      child: createExamsCards(context, exams),
-    );
-  }
-
-  Widget createExamsCards(BuildContext context, List<Exam> exams) {
-    final examCards = <Widget>[
-      DayTitle(
-        day: exams[0].begin.day.toString(),
-        weekDay: exams[0].weekDay,
-        month: exams[0].month,
-      ),
-    ];
-    for (var i = 0; i < exams.length; i++) {
-      examCards.add(createExamContext(context, exams[i]));
-    }
-    return Column(children: examCards);
-  }
-
-  Widget createExamContext(BuildContext context, Exam exam) {
-    final isHidden =
-        Provider.of<ExamProvider>(context).hiddenExams.contains(exam.id);
-    return Container(
-      key: Key('$exam-exam'),
-      margin: const EdgeInsets.fromLTRB(12, 4, 12, 0),
-      child: RowContainer(
-        color: isHidden
-            ? Theme.of(context).hintColor
-            : Theme.of(context).scaffoldBackgroundColor,
-        child: ExamRow(exam: exam, teacher: '', mainPage: false),
+      child: Column(
+        children: [
+          DayTitle(
+            day: exams[0].begin.day.toString(),
+            weekDay: exams[0].weekDay,
+            month: exams[0].month,
+          ),
+          ...exams.map(
+            (exam) => Container(
+              key: Key('$exam-exam'),
+              margin: const EdgeInsets.fromLTRB(12, 4, 12, 0),
+              child: RowContainer(
+                child: ExamRow(exam: exam, teacher: '', mainPage: false),
```

Comment:
Why did the `RowContainer(color)` wrapper disappear?
As I remember, the colour was not determined by the `ExamRow` widget because it differed depending on the widget usage, e.g. in homepage card vs exam page card.